<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableMarks">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>24</y>
      <width>301</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableStudents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>24</y>
      <width>201</width>
      <height>180</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QComboBox" name="subjects">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="actionAddStudent"/>
    <addaction name="actionAddMark"/>
    <addaction name="actionAddSubject"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAdd"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionAddStudent">
   <property name="text">
    <string>Student</string>
   </property>
  </action>
  <action name="actionAddMark">
   <property name="text">
    <string>Mark</string>
   </property>
  </action>
  <action name="actionAddSubject">
   <property name="text">
    <string>Subject</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
